app/controllers/users/personal_keys_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Users
+  class PersonalKeysController < ApplicationController
```

Comment:
I know it's common for controllers to be plural, but I think this would make sense as a singular controller?